mesh/dbus.c

```diff
@@ -167,6 +167,23 @@ bool dbus_append_byte_array(struct l_dbus_message_builder *builder,
 	return true;
 }
 
+bool dbus_append_byte_array2d(struct l_dbus_message_builder *builder,
```

Comment:
I'm not a fan of this function. If anything, this should allow building arrays from `l_queue`. But I'm not sure if this is needed at all. See below.